4-Express/server/controllers/todoController.js

```diff
@@ -0,0 +1,40 @@
+const TodoModel = require('../models/todoModel')
+
+
+const getTodoById = (req, res) => {
+    TodoModel.getTodoById(req.params.id, (todo) => {
```

Comment:
you're not returning the value, does it actually work? 🤔

Reply:
yes. its working using the "res"